app/models/service_provider.rb

```diff
@@ -1,4 +1,9 @@
 class ServiceProvider < ActiveRecord::Base
+  ISSUER_FORMAT_REGEXP = /\Aurn:gov:gsa:(?<protocol>SAML:2\.0|openidconnect)\.profiles:sp:sso:(?<department>.+):(?<app>.+)\z/
```

Comment:
looks like we're getting a rubocop error here still

some options:
1. rubocop ignore comment
2. `/x` regex mode to make it a multi-line regex (https://ruby-doc.org/core-2.1.1/Regexp.html#class-Regexp-label-Free-Spacing+Mode+and+Comments)